subprojects/parseq/src/main/java/com/linkedin/parseq/retry/termination/LimitRate.java

```diff
@@ -0,0 +1,27 @@
+package com.linkedin.parseq.retry.termination;
+
+/**
+ * A termination policy that limits the rate of retries.
```

Comment:
IMO we should add an brief introduction on how this rate limiter works (sliding window, leaky bucket, etc) because other two retry policy are quite self-explanary (duration, attempts)

Reply:
Instead of adding it to the interface, I'll add the description to the implementation.